app/services/task_status/validators/proceedings_types.rb

```diff
@@ -0,0 +1,34 @@
+module TaskStatus
+  module Validators
+    class ProceedingsTypes < Base
+      def valid?
+        return false if proceedings.empty?
+
+        super
+      end
+
+    private
+
+      delegate :proceedings, to: :application
+
+      def forms
+        [
+          client_involvement_type_form,
+          delegated_functions_form,
```

Comment:
[ditto](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/7892/files#r2184625932)